jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/QueryEditorDialog.kt

```diff
@@ -53,10 +53,20 @@ class QueryEditorDialog(
         view.queryLogGroupsRadioButton.addActionListener {
             view.queryBox.isEnabled = true
             view.querySearchTerm.isEnabled = false
+            view.saveQueryButton.isEnabled = true
         }
         view.searchTerm.addActionListener {
             view.queryBox.isEnabled = false
             view.querySearchTerm.isEnabled = true
+            view.saveQueryButton.isEnabled = false
+        }
+
+        view.saveQueryButton.addActionListener {
+            if (view.queryBox.text.isNotEmpty()) {
```

Comment:
Could replace this with inline if:
```kt
val text = view.queryBox.text
SaveQueryDialog(project, if(text.isNotEmpty()) {text} else {default_query}, logGroupNames).show()
```
or 
```kt
val text = if(view.queryBox.text.isNotEmpty()) { view.queryBox.text } else { default_query}
SaveQueryDialog(project, text, logGroupNames).show()
```
it makes it more clear what the if is doing

Reply:
Replaced the if block to inline